src/main/java/com/breskul/hw/endpoints/NasaPictureEndpoints.java

```diff
@@ -0,0 +1,46 @@
+package com.breskul.hw.endpoints;
+
+import com.breskul.hw.controllers.NasaPictureController;
+import com.breskul.hw.endpoints.dto.LargestNasaPictureRequest;
+import lombok.SneakyThrows;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+
+@RestController
+public class NasaPictureEndpoints {
+
+    Logger logger = LoggerFactory.getLogger(NasaPictureEndpoints.class);
```

Comment:
If you already use Lombok in your application, you can also use it to declare a logger using the @Sl4j annotation